src/Globe/TileMesh.js

```diff
@@ -159,10 +161,17 @@ TileMesh.prototype.setBBoxZ = function setBBoxZ(min, max) {
         const trans = this.normal.clone().setLength(delta.y);
 
         this.geometry.boundingSphere.radius = Math.sqrt(delta.x * delta.x + this.oSphere.radius * this.oSphere.radius);
+        this.updateGeometricError();
         this.centerSphere = new THREE.Vector3().addVectors(this.oSphere.center, trans);
     }
 };
 
+TileMesh.prototype.updateGeometricError = function updateGeometricError() {
+    // The geometric error is calculated to have a correct texture display.
+    // For the projection of a texture's texel to be less than or equal to one pixel
+    this.geometricError = this.geometry.boundingSphere.radius / SIZE_TEXTURE_TILE;
```

Comment:
Yes is the same, 

Overall, the geometric error for DMT could be SizeTile / segment Count. 
But with the method, the SSE SUBDIVISION THRESHOLD must be larger, because the projection of a segment will be at the maximum of a pixel. We would then have much more segment to display.

Here, as the images are represented in texels (image + geometry). This calculation considers that a texel must not exceed one pixel.

This calculation of meaning because the other on the master does not make sense.
The best would be to make it parametric